PlatformUIJedio/PlatformUIJedio/Presenters/SettingsViewPresenter.swift

```diff
@@ -157,4 +158,8 @@ open class SettingsViewPresenter: BaseSettingsViewPresenter {
         }
         return textViewModel
     }
+    
+    open func onInputValueChanged(input: FieldInput) {
+        
+    }
```

Comment:
what's this for?

Reply:
So subclass can overwrite with custom behavior.